Makefile

```diff
@@ -37,3 +37,9 @@ import-xml-debug:
 
 import-xml-nodebug:
 	php tests/App/console.php process:run --context filepath=$(FILEPATH) -- import_product_xml
+
+
+worker:
```

Comment:
A voir

Reply:
Il semble néscéssaire de déclarer également la queue au niveau du consumer ou de créer un setup:broker.

Reply:
Il semble que le bind du consumer sur l'exchange ne set pas la routing_key

Reply:
Il semble que le valid sur la dernière itération des messages reste en attente et que donc celui ci n'est pas consommé avant qu'un autre messaga arrive

Reply:
Il faut laisser le choix de requue ou non un message fail

Reply:
Mmh, visiblement les messages passent en unacked ... a vérifier

Reply:
Penser à vérifier que la génération du consumer tag, ne crée par de conflit sinon il faudra un id unique par chanel, en théorie non car on ne mutualise pas un channel de message pour le moment.

Est-ce que la mutualisation de chanel peut être intéréssente pour les optim de perf ?